src/functions/delimsizing.js

```diff
@@ -224,11 +230,15 @@ defineFunction({
         if (hadMiddle) {
             for (let i = 1; i < inner.length; i++) {
                 const middleDelim = inner[i];
-                if (middleDelim.isMiddle) {
+                // Property `isMiddle` not defined on `span`. See comment in
+                // "middle"'s htmlBuilder.
+                // $FlowFixMe
+                const isMiddle: IsMiddle = middleDelim.isMiddle;
```

Comment:
I find it really weird that `isMiddle` is not a boolean... we should probably change this property to be called simply `middle`.  Maybe add a TODO to make that change separately.